frontend/public/actions/k8s.ts

```diff
@@ -112,7 +112,7 @@ export const watchK8sObject = (
 
   const { subprotocols } = getState().UI.get('impersonate', {});
 
-  WS[id] = k8sWatch(k8sType, { ...query, subprotocols }).onbulkmessage((events) =>
```

Comment:
this looks like a bug too. `subprotocols` were never passed properly to websocket